### DISCUSSION TITLE
Ender 3 v2 Sprite Extruder Homing Problem

### DISCUSSION THREAD

**Discussion by ozdenfurkan1** [mriscoc/Ender3V2S1#748] [Q&A]
Hi there,

I had dual z and crtouch on my ender 3 v2 (4.2.2). I was add sprite extruder pro last day. I installed this version as you suggested (Ender3V2-422-BLTUBL-T13-MPC-20221222.bin)

But it no longer starts from the middle of the table like it used to. My english is not good  but you can see on this video: https://youtube.com/shorts/yMhPgIuAiPc?feature=share

Since the homing process does not start right in the middle, cr touch touches the outside in this way. 

In the original extruder, the crtouch was further back. I guess that's the problem. But I haven't seen anyone having this problem.

I installed firmware for sprite extruder pro from Creality's site. And it works seamlessly with it. But I would love to use your firmware. I am not at all satisfied with the original firmware

**Comment by fuzelet** [comment]
Did you update the probe settings to account for the location of the probe?

https://github.com/mriscoc/Ender3V2S1/wiki/Probe-Settings-Menu

**Reply by ozdenfurkan1** [reply]
Do you know what numbers should I write for the sprite extruder?

**Comment by mriscoc** [comment]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
